From eb07fa80e3cf972453b61ec957179b1a412713c3 Mon Sep 17 00:00:00 2001
From: Adam Borowski <kilobyte@angband.pl>
Date: Sat, 3 Dec 2016 10:22:52 +0100
Subject: [PATCH] Make all of non-ASCII word-chars, for consistency.

With everything above U+00FF selected, it makes no sense to exclude a few of
latin-1 digits and symbols.
---
 src/selection.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/selection.c b/src/selection.c
index 29dba14..30ddb93 100644
--- a/src/selection.c
+++ b/src/selection.c
@@ -127,10 +127,10 @@ void set_lut(const char *def)
     0x03FFE000, /* digits and "-./"  */
     0x87FFFFFE, /* uppercase and '_' */
     0x07FFFFFE, /* lowercase         */
-    0x00000000,
-    0x00000000,
-    0xFF7FFFFF, /* latin-1 accented letters, not multiplication sign */
-    0xFF7FFFFF  /* latin-1 accented letters, not division sign */
+    0x00000000, /* control chars     */
+    0xFFFFFFFE, /* latin-1 symbols   */
+    0xFFFFFFFF, /* latin-1 accented letters, multiplication sign */
+    0xFFFFFFFF  /* latin-1 accented letters, division sign */
   }; /* all of Unicode above U+00FF is considered "word" chars, even
         frames and the likes */
 
-- 
2.10.2

